modules/update-service-mirror-release.adoc

```diff
@@ -0,0 +1,168 @@
+[id="update-service-mirror-release_{context}"]
+
+[IMPORTANT]
+====
+To avoid excessive memory usage by the OpenShift Update Service application the release images should be mirrored to a separate repository. To accomplish this you must follow the instructions below.
+====
```

Comment:
The module should have an intro sentence, so consider unwrapping this text and using the wording for the intro, such as:

> To avoid excessive memory usage by the OpenShift Update Service application, it is recommended that you mirror release images to a separate repository, as described in the following procedure.

Not sure if "should" = "must" or "is recommended", so suggest changing to "you must mirror..." if that's more accurate.

Reply:
Title and intro sentence added and made minor tweaks to the admonition (using "it is recommended" verbiage).